.gitignore

```diff
@@ -12,6 +12,9 @@
 # backend
 /bin
 
+# api
+/dist
```

Comment:
I think the `/dist` path should also be in the `.prettierignore`. (And probably `package.json`'s `files` array.)

Reply:
Added to `.prettierignore`; thanks! I had deliberately omitted this from
`"files"` in `package.json` because this should not be included when
this package is installed as a dependency, because it is unstable and
only used for ad hoc testing and exploration in Observable notebooks.